whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/transform/FilterAuthFunction.java

```diff
@@ -46,13 +47,18 @@ public class FilterAuthFunction implements FilterFunction {
     public static final String FILTERED_APPENDIX = " # Filtered";
 
     private List<String> passwords = null;
+    private String override;
+    private boolean isTrusted;
     private OAuthSession oAuthSession;
     private UserSession  userSession;
     private RpslObjectDao rpslObjectDao = null;
+    private UserDao userDao;
     private AuthServiceClient authServiceClient;
     private List<X509CertificateWrapper> certificates;
     private ClientAuthCertificateValidator clientAuthCertificateValidator;
 
+    private static final Splitter OVERRIDE_SPLITTER = Splitter.on(',').trimResults().limit(3);
```

Comment:
I guess you're copying from `OverrideCredential` ? Maybe move the Override class to the common package?

Reply:
You could leave the OverrideCredential in update but create an Override domain class in commons.

Reply:
To be honest I didn't want to do any refactor here. I can go with a little one if needed yes to avoid duplicated logic

Reply:
I think we should use overrideOptiosn class and a method that takes string

Reply:
If we support authenticated queries , the whole credential package should be in commons and this way it would be easy to keep the authentication method at one place and easy to maintain and consistent. But I guess that requires some refactor seperate from this PR.Lets have a discussion

Reply:
I'd prefer not to expose internal logic like the splitter used for handling override. Better to move or create an Override class in the commons module and use it both in query and override.

Reply:
Can you shift OverrideValues in OverrideCredential to commons, there is function that parses the string value into OverrideValues, this shoudl also shift

Reply:
this is done as part of 
https://github.com/RIPE-NCC/whois/pull/1751
and 
https://github.com/RIPE-NCC/whois/pull/1750